src/core/Menu/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Menu as RawMenu, MenuProps, PopoverOrigin } from "@material-ui/core";
+import React from "react";
+
+const ANCHOR_ORIGIN: PopoverOrigin = {
+  horizontal: "center",
+  vertical: "bottom",
+};
+
+const TRANSFORM_ORIGIN: PopoverOrigin = {
+  horizontal: "center",
+  vertical: "top",
+};
+
+export { MenuProps };
+
+const Menu = (props: MenuProps): JSX.Element => {
```

Comment:
Menu needs to open at the bottom of the button instead of the default overlapping position